Handle dictionaries in zstd

kampersanda opened the pull request:
This PR modifies all CLI tools to handle zstd compressed dictionaries.

### Note 1

Although `ruzstd` was used in `dictgen`, this modification unifies the dependency to `zstd` throughout this repository because confusion can arise when different libraries with similar functions are used. Pure Rust would be preferable, but the advantages of unifying into one dependency would be greater.

### Note 2

This PR adds shell hints to code blocks to highlight comment-outs.

vbkaisetsu commented on README.md:
````diff
@@ -43,24 +43,26 @@ First of all, install `rustc` and `cargo` following the [official instructions](
+```shell
+# Specify an appropriate release tag to VERSION (e.g., v0.4.0).
+$ wget https://github.com/daac-tools/vibrato/releases/download/VERSION/ipadic-mecab-2_7_0.tar.gz
 $ tar -xzf ipadic-mecab-2_7_0.tar.gz
````
Could you use `.tar.xz`?

kampersanda replied:
Sure. https://github.com/daac-tools/vibrato/pull/112/commits/1dba2931858257e83e322bdedfe0c1f636d726c4

vbkaisetsu commented on README.md:
````diff
@@ -95,7 +97,7 @@ but with the default settings it can produce different tokens from MeCab.
 For example, MeCab ignores spaces (more precisely, `SPACE` defined in `char.def`) in tokenization.
 
-```
+```shell
````
This block contains the result of the command, and that result is highlighted incorrectly.
Blocks should be left as is.

kampersanda replied:
I reverted it for code blocks containing output results. https://github.com/daac-tools/vibrato/pull/112/commits/8cdc1ec4d8d3720f3ade4b9ae5b2dc12b30eb41d

vbkaisetsu commented on README.md:
````diff
@@ -43,24 +43,26 @@ First of all, install `rustc` and `cargo` following the [official instructions](
+```shell
+# Specify an appropriate release tag to VERSION (e.g., v0.4.0).
+$ wget https://github.com/daac-tools/vibrato/releases/download/VERSION/ipadic-mecab-2_7_0.tar.xz
+$ tar -xzf ipadic-mecab-2_7_0.tar.xz
````
```suggestion
$ tar xf ipadic-mecab-2_7_0.tar.xz
```

vbkaisetsu reviewed (commented):
It would be better to revert all code blocks for consistency.

kampersanda commented:
@vbkaisetsu The reason why using `shell` syntax is to highlight comments in code blocks. So, I moved comments outside and reverted them. https://github.com/daac-tools/vibrato/pull/112/commits/48aefebcdd8a70901bd435d296569faabd77a4db
